vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/style/Style.java

```diff
@@ -64,4 +65,16 @@ public void setStroke(Stroke stroke) {
         this.stroke = stroke;
         addChild(stroke);
     }
+
+    @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+    @JsonIdentityReference(alwaysAsId = true)
+    public Text getText() {
+        return text;
+    }
+
+    public void setText(Text text) {
+        removeChild(this.text);
+        this.text = text;
+        addNullableChild(text);
+    }
```

Comment:
Missing javadoc.

Reply:
Thanks, added.

Good point about the other missing documentation. We can try to do this as a follow-up maybe. Though I feel constructor docs are not that urgent, and Fill, Stroke are somewhat self-explanatory.